docs/tutorials/platform_beginner/7_customize_login_and_register_forms.md

```diff
@@ -0,0 +1,146 @@
+# Step 7 - Customize the User forms
+
+We want to have a consistent design over our web site. When using the pagelayout for the user forms, you see the design of your website. We want to welcome our new users with a pretty register page.
+
+## Use custom template for user forms
```

Comment:
I'd say `user registration forms` or, short, `register forms` throughout the whole text, because now it sounds like two different types of forms: user and register